### ISSUE TITLE
problem finding fftTest module

### ISSUE THREAD

**Issue by venkatnarendra** [jackietom/Speaker-Recognition#1]
how to install fftTest module ?

**Comment by jackietom** [comment]
Hi, I'm sorry that I have changed the name of 'fftTest' to 'fft' while leaving fftTest in other files unchanged. 
I have changed it just now. Thank you for your question.

**Comment by venkatnarendra** [comment]
hii, i am a newbie , i should thank you for the code :)

what changes should i make, to use your code ... i mean which folders to create and what to add them
if you don't mind can you explain me working of this program

i will be very thankfull to you :)

**Comment by jackietom** [comment]
Actually I write this code for my "signals and systems" course to finish my big homework. It can be divided into two parts. The first method is very brute-force: getting the fft of the test sample and compare it with data in the database (by doing inner product). the second method is using MFCC method (you can read some books about it)
  if you want to run the code, you can change the PATH of the database and samples in MFCC.py (MFCC method) and audioTest.py (fft method). I think you can find them.
  By the way, The code is just a homework. if you are tying to make some products I suggest you to 
search some papers and other professional materials.

**Comment by venkatnarendra** [comment]
thank you for the reply , i want to implement speaker recognition in python so i searched git-hub and found your code . I want to try your code , i even added folders named enroll and test in folder which contains python script and changed the path to that folders but still not working . and also i want to know the way how to make use of your python script to recognize speakers
the flow it works

**Comment by jackietom** [comment]
The resources I used is openslr:data_thuyg20_sre.tar.gz  (http://www.openslr.org/22/). I used part of its files and I cut the audio to be stored in database into 30s long and the audio used to test into '10s'. the 'enroll' folder is part of the database and I used F101_train.wav (30s long) in enroll folder as the audio to be put into the database and used F101_test_(1 to 17).wav in the test folder (10s each) to test. I remember I can just run the MFCC.py and get the results if all the data is in proper place.  I think your code won't work maybe caused by the lengths of the audios are not cut into '10s' and '30s'
